contracts/contracts/VerificationGateway.sol

```diff
@@ -16,12 +17,13 @@ The gateway holds a single ProxyAdmin contract for all wallets, and can
 only called by a wallet that the VG created, and only if the first param
 is the calling wallet's address.
  */
-contract VerificationGateway
+contract VerificationGateway is EIP712
 {
-    /** Domain chosen arbitrarily */
-    bytes32 BLS_DOMAIN = keccak256(abi.encodePacked(uint32(0xfeedbee5)));
     uint8 constant BLS_KEY_LEN = 4;
 
+    string public constant BLS_DOMAIN = "BLS_WALLET";
```

Comment:
#nitpick, prefer something like `BLS_DOMAIN_NAME`, as this is not the actual domain.

```suggestion
    string public constant BLS_DOMAIN_NAME = "BLS_WALLET";
```